community.py

```diff
@@ -1069,6 +1069,12 @@ def unload_community(self):
 
         self.dispersy.detach_community(self)
 
+    def is_loaded(self):
+        """
+        Is this community attached to Dispersy
```

Comment:
Comment looks a bit weird, I would propose something like `Returns whether this community is attached to Dispersy`.